doc/source/data/api/grouped_dataset.rst

```diff
@@ -0,0 +1,34 @@
+
+GroupedDataset API
+==================
+
+.. autoclass:: ray.data.grouped_dataset.GroupedDataset
+    :members:
+
+Aggregations
+------------
+
+.. autoclass:: ray.data.aggregate.AggregateFn
+    :members:
+
+.. autoclass:: ray.data.aggregate.Count
+    :members:
+
+.. autoclass:: ray.data.aggregate.Sum
+    :members:
+
+.. autoclass:: ray.data.aggregate.Max
+    :members:
+
+.. autoclass:: ray.data.aggregate.Mean
+    :members:
+
+.. autoclass:: ray.data.aggregate.Std
+    :members:
+
+.. autoclass:: ray.data.aggregate.AbsMax
+    :members:
+
+
+Map transformations
+-------------------
```

Comment:
I'm assuming that this isn't needed since the group transformation APIs are all on `GroupedDataset`.
```suggestion
```

Reply:
Oops